### PR TITLE
Use perma links on nodes tree

### PR DESCRIPTION
**Pull request by roggervalf**
#### Changes Made
This PR adds perma links on tree view, so we can easily go to an specific job details page from tree view.
#### Potential Risks
No potential risks
#### Test Plan
1. Go to example directory
2. Run npm run start:bullmq_with_flows
3. Go to the first flow connection
4. Create a flow, for example using:
```
{
  "name": "parent-job",
    "queueName": "name_of_my_parent_queue",
    "data": {},
    "children": [
      {"name": "child","data": {"idx": 0, "fo": "bar"}, "queueName":"name_of_my_queue"}
    ]
}

```
5. You should see the tree view and if you click to the name of each node, you should go to that job detail
6. Also if you look for that flow using, same behaviour of perma links should work.
#### Checklist

- [ ] I've increased test coverage
- [x] Since this is a public repository, I've checked I'm not publishing private data in the code, commit comments, or this PR.

### CODE REVIEW

**Comment by roggervalf** [comment]
Please @bradvogel when you get a chance, some 👀 on this pr.
Thank you very much

File: public/dashboard.js [hunk 1]
```diff
@@ -5,18 +5,28 @@ $(document).ready(() => {
...
+
+    const href = `${basePath}/${encodeURIComponent(
+      flowHost
+    )}/${encodeURIComponent(job.queueName)}/${job.id}`;
 
     if (children && children.length > 0) {
       return {
         text,
-        nodes: children.map((child) => formatToTreeView(child)),
+        // href,
```
Comment (bradvogel) [inline public/dashboard.js:23]: mind removing commented out code

Comment (roggervalf) [reply public/dashboard.js]: sorry about that, I addressed it in last commit

**Review by bradvogel — APPROVED** [review]

**Pull request merged**